<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetAxisTitlesDialog</class>
 <widget class="QDialog" name="SetAxisTitlesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Set axis titles</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" colspan="4">
      <widget class="QLabel" name="ql_yAxis">
       <property name="text">
        <string>Y axis</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="ql_xUnit">
       <property name="text">
        <string>Unit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="qle_xType"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="qle_yType"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ql_xType">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="qle_yUnit"/>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="ql_xAxis">
       <property name="text">
        <string>X axis</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="ql_yUnit">
       <property name="text">
        <string>Unit</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ql_yType">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="qle_xUnit"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="qpb_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qpb_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>qle_xType</tabstop>
  <tabstop>qle_xUnit</tabstop>
  <tabstop>qle_yType</tabstop>
  <tabstop>qle_yUnit</tabstop>
  <tabstop>qpb_ok</tabstop>
  <tabstop>qpb_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
